flow/designs/asap7/mock-array/config.py

```diff
@@ -27,19 +28,12 @@
 margin_y    = placement_grid_y
 
 # Element core margin
-ce_margin_x = placement_grid_x * 0.5
-ce_margin_y = placement_grid_y * 0.5
-
-# PDN problems if it is smaller. Not investigated.
-array_spacing_x = margin_x * 2
-array_spacing_y = margin_y * 2
-
-array_offset_x = array_spacing_x + margin_x
-array_offset_y = array_spacing_y + margin_y
+ce_margin_x = placement_grid_x * 0.25
```

Comment:
use only integers, change placement_grid to be smallest legal value and multiply to get integer.multiples